### PR TITLE
Fixed redundant codes and typo in views

### PR DESCRIPTION
**Pull request by thangchung**
I just walk through the source code and find some redundant codes and typos so I fixed them.

### CODE REVIEW

File: src/Web/Views/Shared/_Footer.cshtml [hunk 1]
```diff
@@ -7,7 +7,7 @@
     </div>
 
     <div class="column is-half">
-      <nav class="navbar is-transparent" role="navigation" aria-label="footer navigation">
+      <nav class="navbar is-transparent" role="navigation" aria-label="footer navigation"></nav>
```
Comment (bacongobbler) [inline src/Web/Views/Shared/_Footer.cshtml:10]: I think the closing tag needs to be placed after line 16. Notice the divs and unordered lists on the next few lines are marked as `navbar-end` and `navbar-item`, respectively. Those should be children of the `nav` tag.

https://bulma.io/documentation/components/navbar/

**Review by bacongobbler — CHANGES_REQUESTED** [review]

**Comment by bacongobbler** [comment]
Thanks! Just found something that needs to be fixed, but the rest of the changes look good.

File: src/Web/Views/Shared/_Footer.cshtml (hunk 1 above)
Comment (thangchung) [reply src/Web/Views/Shared/_Footer.cshtml]: Okay. I just modified it

**Review by bacongobbler — APPROVED** [review]

**Comment by bacongobbler** [comment]
thanks!

**Pull request merged**